R/plotBRICKcalib.R

```diff
@@ -0,0 +1,64 @@
+#' Render the BRICK calibration plotting routine
+#'
+#' Renders the file plotCalibration.Rmd to create the plots for the BRICK calibration
+#'
+#' @param outputDir (named) character vector, path(s) to output directories.
+#'  If several paths are given, the names can be used to pass short scenario names.
+#' @param cal character vector, name(s) of file(s) with calibration reporting results
+#' @param outName character, string added to the pdf file name and names of additionally saved plots
+#' @param scenNames character vector, scenario names for different paths.
+#'  Needs to be specified if \code{outputDir} is unnamed and contains more than one element.
+#' @param savePlots logical, whether all plots should additionally be saved as png
+#'
+#' @author Ricarda Rosemann
+#'
+#' @importFrom piamutils getSystemFile
+#' @importFrom rmarkdown render
+#' @export
+
+plotBRICKCalib <- function(outputDir = ".", cal = "BRICK_calibration_report.csv",
+                           outName = "", scenNames = NULL,
+                           savePlots = FALSE) {
+
+  # Extract the scenario name from the output directory
+  scenario <- sub("_\\d{4}-\\d{2}-\\d{2}_\\d{2}.\\d{2}.\\d{2}", "", basename(outputDir))
```

Comment:
Minor detail: escape the dot.
```suggestion
  scenario <- sub("_\\d{4}-\\d{2}-\\d{2}_\\d{2}\\.\\d{2}\\.\\d{2}", "", basename(outputDir))
```